Packs/DeprecatedContent/pack_metadata.json

```diff
@@ -2,7 +2,7 @@
     "name": "Deprecated Content",
     "description": "Deprecated Cortex XSOAR content pack.",
     "support": "xsoar",
-    "currentVersion": "1.6.7",
```

Comment:
@evisochek Wasnt this suppose to be removed?

Reply:
removed now